Libraries/LibGUI/FileSystemModel.h

```diff
@@ -47,6 +47,7 @@
     __ENUMERATE_FILETYPE(pdf, ".pdf")          \
     __ENUMERATE_FILETYPE(python, ".py")        \
     __ENUMERATE_FILETYPE(sound, ".wav")        \
+    __ENUMERATE_FILETYPE(markdown, ".md")      \
     __ENUMERATE_FILETYPE(ini, ".ini")          \
     __ENUMERATE_FILETYPE(text, ".txt", ".md")
```

Comment:
```suggestion
    __ENUMERATE_FILETYPE(text, ".txt")
```

I initially added this to have Markdown files show up as regular text files, but we don't need that anymore :)